glean-core/tests/ping_maker.rs

```diff
@@ -88,7 +88,6 @@ fn collect_must_report_none_when_no_data_is_stored() {
 }
 
 #[test]
-#[ignore] // sequence numbers aren't implemented
 fn seq_number_must_be_sequential() {
```

Comment:
nit: can we slightly augment this test so that after the store increase together one store keeps increasing? This is to test that we don't return the same sequences for all the stores :)